api-metastore/src/test/java/org/zalando/nakadi/controller/EventTypeControllerTest.java

```diff
@@ -239,17 +236,6 @@ public void whenPostWithNoSchemaSchemaThenReturn422() throws Exception {
         postETAndExpect422WithProblem(eventType, expectedProblem);
     }
 
-    @Test
```

Comment:
All of these tests are related to schema change and they were moved to the appropriate layer under `SchemaServiceTest`. I didn't add anything new here.